tests/pytorch/test_numerics.py

```diff
@@ -1073,7 +1073,8 @@ def test_layernorm_linear_accuracy(dtype, bs, model, normalization, zero_centere
 @pytest.mark.parametrize("model", model_configs.keys())
 @pytest.mark.parametrize("activation", all_activations)
 @pytest.mark.parametrize("normalization", all_normalizations)
-def test_layernorm_mlp_accuracy(dtype, bs, model, activation, normalization):
+@pytest.mark.parametrize("ub_split_ag", all_boolean)
+def test_layernorm_mlp_accuracy(dtype, bs, model, activation, normalization, ub_split_ag):
```

Comment:
Looking closer, this test doesn't enable FP8, so it isn't actually testing anything meaningful. I don't think the testing infrastructure is currently in a state to properly write small isolated tests, and making those changes is outside the scope of this PR. I propose we revert these changes and let you vouch for correctness in your own use-case. This is an expert feature anyways. We should revisit after refactoring these tests. 

```suggestion
def test_layernorm_mlp_accuracy(dtype, bs, model, activation, normalization):
```

Related: https://github.com/NVIDIA/TransformerEngine/pull/510, https://github.com/NVIDIA/TransformerEngine/pull/503